content/docs/get-started/policy-as-code/_index.md

```diff
@@ -1,28 +1,26 @@
 ---
-title: Get Started with Policy as Code (Preview)
+title: Get Started with Policy as Code
 linktitle: Policy as Code
 menu:
   getstarted:
     identifier: pac
     weight: 3
 ---
-{{% pac-preview %}}
+{{% crossguard-preview %}}
 
-Pulumi has an exciting new feature which provides Gated Deployments via Policy as Code.
+Pulumi has an exciting new product, CrossGuard, which provides gated deployments via Policy as Code.
 
-Often organizations want to empower developers to manage their infrastructure yet are concerned about giving them full access. Policy as Code (PaC) allows administrators to provide autonomy to their developers while ensuring compliance to defined organization policies.
+Often organizations want to empower developers to manage their infrastructure yet are concerned about giving them full access. CrossGuard allows administrators to provide autonomy to their developers while ensuring compliance to defined organization policies.
 
-With this new feature, users can express business or security rules as functions that are executed against resources in their stacks. When policies are executed as part of your Pulumi deployments, any violation will gate or block that update from proceeding.
-
-Organization administrators can write and publish Policy Packs for their organization. These Policy Packs can be enforced against different Policy Groups or groups of stacks.
+Using Policy as Code, users can express business or security rules as functions that are executed against resources in their stacks. Then using CrossGuard, organization administrators can apply these rules to particular stacks within their organization. When policies are executed as part of your Pulumi deployments, any violation will gate or block that update from proceeding.
 
 ## Terminology
 
 * **Policy Pack** - a set of related policies - i.e. “Security”, “Cost Optimization”, “Data Location”. The categorization of policies into a policy pack is left up to the user.
 * **Policy** - an individual policy - i.e. “prohibit use of instances larger than t3.medium”.
 * **Enforcement Level** - the impact of a policy violation - i.e. “mandatory” or “advisory”.
 
-You can learn more about the core concepts of Policy as Code [here]({{< relref "core-concepts" >}}).
+You can learn more about the core concepts of Policy as Code [here]({{< relref "/docs/guides/crossguard/core-concepts" >}}).
```

Comment:
For SEO, avoid links like "[here]". Instead, put more meaningful text in the anchor link like "Learn more about [Policy as Code core concepts]."